pkgs/development/tools/pipenv/default.nix

```diff
@@ -47,7 +47,7 @@ in buildPythonApplication rec {
     # and to call setup.py.
     # It would use sys.executable, which in our case points to a python that
     # does not have the required dependencies.
-    substituteInPlace pipenv/core.py \
+    substituteInPlace pipenv/utils/virtualenv.py \
```

Comment:
For such an important package it would be good to have a test which verifies that this sort of patching has done the right thing.